api/src/tests/accounts_test.rs

```diff
@@ -150,42 +149,43 @@ async fn test_get_account_resources_by_invalid_ledger_version() {
     context.check_golden_output(resp);
 }
 
+// TODO(George): FIXME
 // figure out a working module code, no idea where the existing one comes from
-#[ignore] // TODO(issue 81): re-enable after cleaning up the compiled code in the test
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-async fn test_get_account_modules_by_ledger_version() {
-    let mut context = new_test_context(current_function_name!());
-    let code = "a11ceb0b0300000006010002030205050703070a0c0816100c260900000001000100000102084d794d6f64756c650269640000000000000000000000000b1e55ed00010000000231010200";
-    let root_account = context.root_account().await;
-    let txn = root_account.sign_with_transaction_builder(
-        context
-            .transaction_factory()
-            .module(hex::decode(code).unwrap()),
-    );
-    context.commit_block(&vec![txn.clone()]).await;
-    let modules = context
-        .get(&account_modules(
-            &context.root_account().await.address().to_hex_literal(),
-        ))
-        .await;
-
-    assert_ne!(modules, json!([]));
-
-    let modules = context
-        .get(&account_modules_with_ledger_version(
-            &context.root_account().await.address().to_hex_literal(),
-            0,
-        ))
-        .await;
-    assert_eq!(modules, json!([]));
-}
-
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-async fn test_get_core_account_data() {
-    let mut context = new_test_context(current_function_name!());
-    let resp = context.get("/accounts/0x1").await;
-    context.check_golden_output(resp);
-}
+// #[ignore] // TODO(issue 81): re-enable after cleaning up the compiled code in the test
```

Comment:
@banool @gregnazario do you know what is this issue? Do we want to kill this test or keep it? cc @zekun000 who wrote the comment I believe :)

Reply:
This API test framework is incomplete.

We likely want to write an e2e test to replace it (which would be good to add)

Can you move something similar, where you publish a contract and view the modules in `smoke-test/api`

Reply:
I don't think we need to push towards smoke tests. They are both more complicated and take a lot longer to execute.

The issue here was we didn't have a convenient way to compile, publish, test, so we just disabled. I think you've fixed that. We also don't need so many duplicate tests. I think this is sufficiently handled in the e2e-move-tests so I would just delete it.